core/shared/src/main/scala/fs2/fs2.scala

```diff
@@ -1,5 +1,9 @@
 package object fs2 {
 
+  /** A stream transformation
+    */
+  type VarPipe[-F[_], +G[_], -I, +O] = Stream[F, I] => Stream[G, O]
```

Comment:
I don't want to provide this type alias. In general, we need less type aliases instead of more.

Reply:
Oh, this is something I forgot behind when I was trying to fix a failing test in `TextSpec`, which was caused by the use of _covariance_ functions. I tried defining a `PureIn`, the opposite of `Pure` (an alias for `Any`) to type streams that appeared as inputs... but that did not work well. 

This is a matter for which we should also have some design discussions: how should we define Pipes by default if the `F` is co-variant.